tests/processes/BearRunningTest.py

```diff
@@ -214,8 +214,7 @@ def test_strange_bear(self):
                              LOG_LEVEL.ERROR,
                              LOG_LEVEL.DEBUG,
                              LOG_LEVEL.DEBUG,
-                             LOG_LEVEL.WARNING]
-
```

Comment:
I think you just had trailing whitespaces on that line. If possible please readd it again :)